docs/UserGuide.md

```diff
@@ -9,32 +9,57 @@ title: User Guide (v1.3)
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Introduction
+## Introduction - Overview of the application
```

Comment:
I think this line should be in a new line, so that the header is just "Introduction".  Although, this new line feels kind of redundant since introduction is self-explanatory.

Reply:
Ok sure thank you.I have moved this below